src/controllers/transactionsController.ts

```diff
@@ -193,7 +193,7 @@ export class TransactionsController {
                 const from = this.getAccount(action.fromAddress);
                 const to = this.getAccount(action.toAddress);
                 const quantity = `${amount.toFixed(asset.Accuracy)} ${asset.AssetId}`;
-                const memo = action.toAddress.split(ADDRESS_SEPARATOR)[1] || "";
+                const memo = action.toAddress.split(ADDRESS_SEPARATOR)[1] || operationId;
```

Comment:
what if toAddress contains no separator, will [1] throw NRE exception?

Reply:
No, it will not. In JS it just returns undefined/null.